WTWSMS/events/WtWSMS_zusk_misc_events.txt

```diff
@@ -537,7 +550,7 @@ character_event = {
 			trait = sympathy_christendom
 		}
         modifier = {
-			factor = 5
+			factor = 1.05
```

Comment:
Non-Hellenics and non-Christians living in Roman-controlled areas should not miss out on missing Rome simply because they are not Hellenic or Christian. You don't think there were people in North Africa that missed Rome besides Christians?